assets/style.css

```diff
@@ -94,3 +94,120 @@ h1 {
   background-size: cover;
   margin-top: 30px;
 }
+
+.project-section-container {
+  display: flex;
+  flex-direction: column;
+}
+
+h2 {
+  left: 16px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  color: #1a2236;
+  margin: 39px 16px;
+}
+
+.rectangle_21 img {
+  width: 100%;
+}
```

Comment:
- Hey, great job making the changes requested by the previous reviewer, however, the issue with the slight spaces on the images [in this review summary](https://github.com/axcell-0/Thierrys-portfolio/pull/2#discussion_r1213239261). To improve on this to not make it obvious, kindly add a display property of the block and your design will be better.